### ISSUE TITLE
Other feedback

### ISSUE THREAD

**Issue by NimG98** [OpenLiberty/draft-guide-cloud-openshift#16]
**What you’ll learn**

- [x] Intro cloud paragraph needs to be changed. Refer to this issue to see what needs to be changed: https://github.com/OpenLiberty/guides-common/issues/317

**Prerequisites**

- [x] Different output for `oc version` command than the one on the guide

    ```
    Client Version: version.Info{Major:"4", Minor:"1+", GitVersion:"v4.1.14", 
    GitCommit:"ed73a65c7", GitTreeState:"clean", BuildDate:"2019-08-27T04:36:57Z", 
    GoVersion:"go1.11.5", Compiler:"gc", Platform:"darwin/amd64"}
    Server Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.3", 
    GitCommit:"5e53fd6bc17c0dec8434817e69b04a25d8ae0ff0", GitTreeState:"clean", 
    BuildDate:"2019-06-06T01:36:19Z", GoVersion:"go1.12.5", Compiler:"gc", 
    Platform:"linux/amd64"}
    ```

**Accessing an OpenShift cluster**

- [x] the first `>` looks weird in the sentence:
    > To login to OpenShift using the CLI, navigate to the online web console `> [username] > Copy Login Command > Display Token > Log in` with this token.
 
  so change it to this maybe:

    > To login to OpenShift using the CLI, open the online web console and then navigate to `[username] > Copy Login Command > Display Token > Log in` with this token.

**Deploying microservices to OpenShift**

- [x] "If you’re unfamiliar with Dockerfiles, check out the Using Docker containers to develop microservices guide." should point to the new Containerize docker guide instead

- [x] I don't think we should be using the `dockerfile-maven` plugin, since all of the cloud guides were changed to not use it anymore
    - [x] Instead, manually build the docker images using the `docker build` commands
    - [x] remove all mentions of `dockerfile-maven` in paragraphs

- [x] `MAC & LINUX` is usually `MAC | LINUX` in other guides

**Making requests to the microservices**

- [x] output of registry address is different than the one listed in the guide:
    ```
    default-route-openshift-image-registry.apps.us-west-1.starter.openshift-online.com
    ```
- [x] url for sample hostname is different:
    - original: `inventory-route-[project-name].apps.[region].online-starter.openshift.com`
    - new: `http://inventory-route-nim-project.apps.us-west-1.starter.openshift-online.com`

**Testing the microservices**

- [x] `mvn verify` not working for inventory endpoint test:

    ```
    InventoryEndpointTest.testSuite:72->testHostRegistration:101 The inventory should have one entry for null expected:<1> but was:<0>
    ```

**Other**

- [x] This issue needs to be fixed before publishing: https://github.com/OpenLiberty/draft-guide-cloud-openshift/issues/6
    - `mvn verify` on finish directory fails on SystemEndpointTest instead of InventoryEndpointTest like on start directory

**Comment by MaiHameed** [comment]
Fixed b0c2a6e6bad32d00f57228924bcb18ec527d452c

**Comment by MaiHameed** [comment]
Addressed, closing.
